tsconfig.json

```diff
@@ -22,23 +22,24 @@
     "jsx": "react",
     "baseUrl": ".",
     "paths": {
-        "@mattermost/client": ["packages/client/src"],
+        "@mattermost/client/*": ["packages/client/src/*"],
         "@mattermost/components": ["packages/components/src"],
         "@mattermost/types/*": ["packages/types/src/*"],
         "mattermost-redux/*": ["packages/mattermost-redux/src/*"],
         "reselect": ["packages/reselect/src"],
-        "!!file-loader*": ["utils/empty-string"]
+        "!!file-loader*": ["utils/empty-string"],
+        "@e2e-support/*": ["e2e/playwright/support/*"],
+        "@e2e-test.config": ["e2e/playwright/test.config.ts"]
```

Comment:
Intentionally add `@e2e-` suffix to easily identify that these are used for E2E.

Reply:
Yes, since it's not being recognized, especially by the editor from the root folder.

Reply:
I suppose we could look at adding a dedicated TSConfig for VS Code, but I don't think we need to go that far unless we actually start using TypeScript to compile the web app (as opposed to Babel which it uses currently)